test/test-directive.cpp

```diff
@@ -70,6 +70,33 @@ TEST(DirectiveTest, WildcardTest)
     }
 }
 
+TEST(DirectiveTest, LeadingWildcard)
+{
+    std::vector<std::string> examples = {
+        "/test",
+        "/a/test",
+        "/ab/test",
+        "/abc/test",
+    };
+    std::string directive("*/test");
+    Rep::Directive parsed(directive, true);
+    for (auto example : examples)
+    {
+        EXPECT_TRUE(parsed.match(example)) <<
```

Comment:
I'm not a fan of assertions in loops, but this should cover what you want covered.

Reply:
Normally, I wouldn't do this, but the other tests in this suite were already written the same way.

Reply:
Yeah... apologies about that :-/